mgmtsystem_action/views/mgmtsystem_action.xml

```diff
@@ -0,0 +1,194 @@
+                            <t t-if="record.type_action.raw_value and record.type_action.raw_value == 'correction'" t-set="type_action_color">#ffa500</t>
+                            <t t-if="record.type_action.raw_value and record.type_action.raw_value == 'prevention'" t-set="type_action_color">#00ff00</t>
+                            <t t-if="record.type_action.raw_value and record.type_action.raw_value == 'improvement'" t-set="type_action_color">#0000ff</t>
+                            <div t-attf-class="#{kanban_color( type_action_color)} oe_kanban_global_click">
+                                <div class="o_dropdown_kanban dropdown">
+
+                                    <a class="dropdown-toggle btn" data-toggle="dropdown" href="#">
+                                        <span class="fa fa-bars fa-lg"/>
+                                    </a>
+                                    <ul class="dropdown-menu" role="menu" aria-labelledby="dLabel">
+                                        <t t-if="widget.editable">
+                                            <li>
+                                                <a type="edit">Edit</a>
+                                            </li>
+                                        </t>
+                                    </ul>
+                                </div>
+                                <div class="oe_kanban_content">
+                                    <div>
+                                        <strong><field name="name"/></strong>
+                                    </div>
+
+                                    <div class="text-muted">
+                                        <t t-if="record.date_deadline.raw_value and record.date_deadline.raw_value lt (new Date())" t-set="red">oe_kanban_text_red</t>
+                                        <span t-attf-class="#{red || ''}">
+                                            <field name="date_deadline"/>
+                                        </span>
+                                    </div>
+                                    <div class="o_kanban_footer  pull-right">
+                                        <br/>
+                                        <div>
+                                        <img t-att-src="kanban_image('res.users', 'image_small', record.user_id.raw_value)" t-att-title="record.user_id.value" width="24" height="24" class="oe_kanban_avatar"/>
+                                    </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </t>
+                    </templates>
+                </kanban>
+            </field>
+        </record>
+
+        <!-- Management System Action Report Pivot View -->
+
+                <record id="view_report_mgmtsystem_action_pivot" model="ir.ui.view">
```

Comment:
Action Report view should be defined in their own XML file, inside the reports subdirectory.